README.md

```diff
@@ -90,7 +94,46 @@ An example of three clusters configured to use with Submariner would look like t
 
 # Installation
 
-Submariner installation can be done manually using helm charts or by using its operator. Both methods are described here.
+Submariner installation can be done manually using a dedicated tool, `subctl`, or Helm charts, or
+by using its operator. All three methods are described here.
+
+## Installation using subctl
+
+`subctl` is a Submariner-specific tool developed to simplify Submariner installation. It drives
+[the operator](#installation-using-operator). You can see it in action in
+[this demo](https://youtu.be/cInmBXuZsU8).
+
+### Download
+
+`subctl` is maintained in the operator project; binaries are available for download from
+[the operator releases](https://github.com/submariner-io/submariner-operator/releases).
+Download the latest binary for your operating system and architecture, and install it on your path;
```

Comment:
```suggestion
Download the latest binary for your operating system and architecture and install it on your path.
```

Reply:
Losing the comma makes the previous “and” less clear — the two “and”s are different...

Reply:
How about this:

"Download the latest binary for your operating system and architecture, then install it on your path, as the binary has no external dependencies."

Reply:
Right, “then” works better. There’s no connection between the path and the external dependencies, so I changed it to

> Download the latest binary for your operating system and architecture, then install it on your
path. The binary has no external dependencies so you won’t need anything else.